docs/tables/aws_vpc_flow_log/index.md

```diff
@@ -195,17 +208,130 @@ partition "aws_vpc_flow_log" "my_logs_region" {
-  filter = "action <> 'REJECT'"
-
-  source "aws_s3_bucket" {
+partition "aws_vpc_flow_log" "my_logs_region" {
+  source "aws_s3_bucket"  {
     connection  = connection.aws.vpc_logging
     bucket      = "vpc-flow-logs-bucket"
+    file_layout = `AWSLogs/(%{DATA:org_id}/)?%{NUMBER:account_id}/vpcflowlogs/(us-east-1|us-east-2)/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/(%{NUMBER:hour}/)?%{DATA}.log.gz`
+  }
+}
+```
+
+### Collect VPC flow logs exported from CloudWatch to an S3 bucket
+
+Collect VPC flow logs that were exported from CloudWatch to an S3 bucket, where the logs do not include a header line. Since the exported logs contain raw log lines only, a format block is defined to map the layout of each field.
+
+**Important Note**: The field, `export-timestamp`, is not part of the original log event. It is automatically added by AWS during the export process from CloudWatch to S3. This timestamp represents the time when the log was exported, not when the event occurred.
+
+```hcl
+format "aws_vpc_flow_log" "exported_log_format" {
+  layout = "export-timestamp instance-id interface-id pkt-srcaddr pkt-dstaddr pkt-src-aws-service az-id flow-direction start log-status packets protocol srcaddr dstaddr srcport end subnet-id"
+}
+```
+
+```hcl
+partition "aws_vpc_flow_log" "cw_log_group_logs" {
+  source "aws_s3_bucket" {
+    connection = connection.aws.logging_account
+    format     = format.aws_vpc_flow_log.exported_log_format
+    bucket     = "aws-vpc-flow-logs-123456789012-exported"
+    region     = "us-east-1"
+  }
+}
+```
+
+### Collect logs from a CloudWatch log group
+
+Collect VPC flow logs from all log streams in a CloudWatch log group.
+
+```hcl
+partition "aws_vpc_flow_log" "cw_log_group_logs" {
+  source "aws_cloudwatch_log_group" {
+    connection     = connection.aws.logging_account
+    log_group_name = "aws-vpc-flow-logs-123456789012-fd33b044"
+    region         = "us-east-1"
+  }
+}
+```
+
+### Collect VPC flow logs from a CloudWatch log group with custom log format
+
+Collect VPC flow logs from an AWS CloudWatch Log Group where the log format does not include a header line, or the format differs from the default.
+
+```hcl
+format "aws_vpc_flow_log" "custom" {
+  layout = `account-id action az-id bytes dstaddr dstport end flow-direction instance-id interface-id log-status packets pkt-dst-aws-service pkt-dstaddr pkt-src-aws-service pkt-srcaddr protocol region reject-reason srcaddr srcport start sublocation-id sublocation-type subnet-id tcp-flags traffic-path type version vpc-id`
+}
+```
+
+```hcl
+partition "aws_vpc_flow_log" "cw_log_group_logs" {
+  source "aws_cloudwatch_log_group" {
+    connection     = connection.aws.logging_account
+    format         = format.aws_vpc_flow_log.custom
+    log_group_name = "aws-vpc-flow-logs-123456789012-fd33b044"
+    region         = "us-east-1"
+  }
+}
+```
+
+### Collect VPC flow logs from a CloudWatch log group for all ENIs with wildcard stream names
```

Comment:
This title feels long, is there a shorter/more general title we can use to summarize what this example highlights?